### ISSUE TITLE
Installing strong-pm as service not supported on SUSE11

### ISSUE THREAD

**Issue by b-admike** [strongloop/strong-service-install#13]
Machine info: SUSE11 on z 
Linux zsl11n1 3.0.101-0.47.67-default #1 SMP Wed Aug 19 14:12:37 UTC 2015 (265cc32) s390x s390x s390x GNU/Linux

I am trying to install strong-pm as a service using my own user and deploy an app to it, but I am hitting an issue with upstart/systemd

biniam@zsl11n1:~> sudo "PATH=$PATH" sl-pm-install --user biniam -b /home/biniam/testpmserv/ 
--systemd
Writing job...
Service strong-pm installed (/etc/systemd/system/strong-pm.service)
biniam@zsl11n1:~> sudo /usr/bin/systemctl status strong-pm                    
sudo: /usr/bin/systemctl: command not found

biniam@zsl11n1:~> sudo "PATH=$PATH" sl-pm-install --user biniam -b /home/biniam/testpmserv/
biniam's password:
Writing job...
Service strong-pm installed (/etc/init/strong-pm.conf)
biniam@zsl11n1:~> sudo /sbin/initctl start strong-pm
biniam's password:
sudo: /sbin/initctl: command not found

biniam@zsl11n1:/etc/systemd> echo $PATH
/home/biniam/baslpm/I64_32/bin:/usr/local/bin:/usr/bin:/bin:/usr/bin/X11:/usr/X11R6/bin:/usr/games:/usr/lib64/jvm/jre/bin:/usr/lib/mit/bin:/usr/lib/mit/sbin

I have attached the configuration files for the service created by sl-pm-install.

[strong-pm.conf.txt](https://github.com/strongloop/strong-service-install/files/34387/strong-pm.conf.txt)
[strong-pm.service.txt](https://github.com/strongloop/strong-service-install/files/34388/strong-pm.service.txt)

**Comment by rmg** [comment]
SLES11 doesn't have Upstart or systemd, so this is basically a vote for https://github.com/strongloop-internal/scrum-nodeops/issues/148

**Comment by smartmouse** [comment]
Yes, this is specific to SLES11, SLES 12 has systemd support, and I do not think this is a z-system specific issue but a generic SLES 11 issues (even for power)

**Comment by rmg** [comment]
RH6 uses and old version of Upstart, so you'll install with `--upstart 0.6` on those systems.

**Comment by smartmouse** [comment]
@rmg you are right, that works on RH6.x What are major differences with the different pm-install ? Are they any docs u are point me to please? e.g. with systemd    there are journalctl  and systemctl,  

Wiith upstart m there are  sbin/initctl. etc.

**Comment by cgole** [comment]
@smartmouse all the differences should be listed here. If not, we should add it to the docs here

**Comment by smartmouse** [comment]
@rmg, If I use the upstart method, how can I dump all the status?  On the systemd method, I can use the journalctl,  this does not seem to exist on RH6.x

**Comment by rmg** [comment]
I answered elsewhere, but for anyone finding this later..

When using `--upstart=0.6` the Upstart job that gets created sends its stdout/stderr to syslog, which usually defaults to writing out to a file like `/var/log/messages`. The log entries are tagged with the daemon's name as the program name, so you should be able to grep for it or configure syslog/rsyslog/syslog-ng/etc. to redirect these specific log entries somewhere else.

**Comment by smartmouse** [comment]
@rmg, thanks
